client.go

```diff
@@ -243,6 +251,17 @@ func (r *marathonClient) apiCall(method, uri string, body, result interface{}) e
 		return err
 	}
 
+	// Add any basic auth and the content headers
```

Comment:
Can you remove this section and update [apiRequest()](https://github.com/gambol99/go-marathon/blob/master/client.go#L277) instead ... Authorization was moved into a common method due to issues with marathon sse events